react/client/src/styles/utilityStyles.ts

```diff
@@ -0,0 +1,29 @@
+import { makeStyles, createStyles } from '@material-ui/core';
```

Comment:
Ok so I kept these as utility classes for now because unlike the Button component which doesn't have multiple children (just the buttonText), these can have multiple JSX children elements, curious to know if you think this approach works or if I should turn all these into generic React components as well and figure out how to pass those children in